### ISSUE TITLE
Create installer for Plone 5.2.5

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/Installers-UnifiedInstaller#117]
I have made a final release for Plone 5.2.5.
See links in the [community post](https://community.plone.org/t/plone-5-2-5-released/14176).

Can you make an installer please?
Thanks a lot!

I also asked on the [installers list](https://sourceforge.net/p/plone/mailman/plone-installers/?viewmonth=202108) a few days ago, but perhaps an issue in this repo is clearer? I can do that from now on, if you prefer.

**Comment by avoinea** [comment]
@tkimnguyen ?!

**Comment by mauritsvanrees** [comment]
@jensens or @tkimnguyen Can you handle this please?

**Comment by jensens** [comment]
The build is done by GH actions and the installer is an artifact there. I did replace a lot of 5.2.4 by 5.2.5 ans pushed it - this should do it.

**Comment by jensens** [comment]
Artifact is here https://github.com/plone/Installers-UnifiedInstaller/suites/3723270586/artifacts/91102892 and tests are passing see https://github.com/plone/Installers-UnifiedInstaller/actions/runs/1217150327.
Attention: This is a ZIP by created GH Actions that contains the installer ZIP and TGZ.
Some manual test would be nice, any volunteers?

**Comment by mauritsvanrees** [comment]
I tried the tarball, and seems okay, zeocluster with two clients is created and can start.
I saw one 5.2.4 still in the logs, and have committed a fix, but this seems to have had no effect on the actual Plone version used.

New actions have finished and succeeded.
I had trouble locating the artifacts, but finally found it in the [summary of the actions](https://github.com/plone/Installers-UnifiedInstaller/actions/runs/1229219370) when I scroll to the bottom.

I have uploaded the tarball and zip to [launchpad](https://launchpad.net/plone/5.2/5.2.5) and linked to them on https://plone.org/download/releases/5.2.5
